packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/bedrock/create-model-customization-job.ts

```diff
@@ -0,0 +1,491 @@
+      }),
+      new iam.PolicyStatement({
+        actions: ['ec2:CreateNetworkInterface'],
+        resources: [
+          ...vpcConfig.securityGroups.map(
+            (securityGroup) => Stack.of(this).formatArn({
+              service: 'ec2',
+              resource: 'security-group',
+              resourceName: securityGroup.securityGroupId,
+            }),
+          ),
+          ...vpcConfig.subnets.map(
+            (subnet) => Stack.of(this).formatArn({
+              service: 'ec2',
+              resource: 'subnet',
+              resourceName: subnet.subnetId,
+            }),
+          ),
+        ],
+      }),
+      new iam.PolicyStatement({
+        actions: [
+          'ec2:CreateNetworkInterfacePermission',
+          'ec2:DeleteNetworkInterface',
+          'ec2:DeleteNetworkInterfacePermission',
+        ],
+        resources: ['*'],
+        conditions: {
+          ArnEquals: {
+            'ec2:Subnet': vpcConfig.subnets.map(
+              (subnet) => Stack.of(this).formatArn({
+                service: 'ec2',
+                resource: 'subnet',
+                resourceName: subnet.subnetId,
+              })),
+            'ec2:ResourceTag/BedrockModelCustomizationJobArn': [
+              Stack.of(this).formatArn({
+                service: 'bedrock',
+                resource: 'model-customization-job',
+                resourceName: '*',
+              }),
+            ],
+          },
+          StringEquals: {
+            'ec2:ResourceTag/BedrockManaged': 'true',
+          },
+        },
+      }),
+      new iam.PolicyStatement({
+        actions: ['ec2:CreateTags'],
+        resources: [
+          Stack.of(this).formatArn({
+            service: 'ec2',
+            resource: 'network-interface',
+            resourceName: '*',
+          }),
+        ],
+        conditions: {
+          'StringEquals': {
+            'ec2:CreateAction': ['CreateNetworkInterface'],
+          },
+          'ForAllValues:StringEquals': {
+            'aws:TagKeys': [
+              'BedrockManaged',
+              'BedrockModelCustomizationJobArn',
+            ],
+          },
+        },
+      }),
+    ];
+  }
+
+  private renderPolicyStatements(): iam.PolicyStatement[] {
+    const policyStatements = [
+      new iam.PolicyStatement({
+        actions: [
+          'bedrock:CreateModelCustomizationJob',
+          'bedrock:TagResource',
+        ],
+        resources: [
+          this.props.baseModel.modelArn,
+          Stack.of(this).formatArn({
+            service: 'bedrock',
+            resource: 'custom-model',
+            resourceName: '*',
+          }),
+          Stack.of(this).formatArn({
+            service: 'bedrock',
+            resource: 'model-customization-job',
+            resourceName: '*',
+          }),
+        ],
+      }),
+      new iam.PolicyStatement({
+        actions: ['iam:PassRole'],
+        resources: [this._role.roleArn],
+      }),
+    ];
+    return policyStatements;
```

Comment:
In implementing sfn.TaskStateBase, it is necessary to generate iam.PolicyStatement[] and assign it to this.taskPolicies. In this case, I think it would be difficult to utilize the grant methods. What do you think?

Reply:
unfortunately I think the grants aren't going to support these permissions, so we can do it like this.